tests/unit/models/gpflow/test_models.py

```diff
@@ -616,7 +618,7 @@ def test_sparse_variational_model_attribute() -> None:
 
 
 def test_sparse_variational_model_num_data_mixin_supports_subclasses() -> None:
-    class SVGPSubclass(SVGP):
+    class SVGPSubclass(SVGP):  # type: ignore[misc]
```

Comment:
this is now required bcs of new mypy setting in the toml file?

Reply:
Yes. SVGP has no type hints so as far as mypy is concerned SVGPSubclass is just `Any`. The type ignore is us telling it that we're ok with that.